### DISCUSSION TITLE
user/pwd for pre-installed ute Mac image

### DISCUSSION THREAD

**Discussion by tonik173** [cyberbotics/webots_ros2#671] [General]
Anyone knows the user/pwd for the pre-installed [Version 2023.0.2 for Webots R2023a](https://cyberbotics.com/files/ros2/webots_ros2_2023_0_2.utm.zip) utm image? I tried various combinations of ubuntu/webots/humble/ros2.

**Comment by ygoumaz** [comment]
Sorry for the inconvenience, the password should be completely disabled in the VM image. Do you have to enter the password at login or for a super user command? Anyway, the user/pwd combination is `ubuntu/ubuntu`.
